Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/EditingAndDeletingSyncUp-02-code-0011.swift

```diff
@@ -27,9 +27,8 @@ struct SyncUpDetail {
     Reduce { state, action in
       switch action {
       case .alert(.presented(.confirmButtonTapped)):
-        @Shared(.fileStorage(.syncUps)) var syncUps: IdentifiedArrayOf<SyncUp> = []
-        syncUps.remove(id: state.syncUp.id)
-        return .none
+        // @Shared(.fileStorage(.syncUps)) var syncUps: IdentifiedArrayOf<SyncUp> = []
+        @Shared(.syncUps) var syncUps
```

Comment:
📓 Here I've added a new step to suggest the usage of the typesafe shared key instead of the "basic" .fileStorage syntax. And due to that, as with one of my previous PR, the next diffs will be weird due to the shift of the files numbering.